README.md

```diff
@@ -1,2 +1,2486 @@
-# ToDoApp
```

Comment:
could we remove content of this README.md ? We will add here our own readme, but please leave for now everything what is before `Table of Contents`